backend/blueprint/api/controllers/registerUser.py

```diff
@@ -1,21 +1,19 @@
 from flask import Blueprint, jsonify, request
 from firebase_admin import auth
-from api.utils.response import Response
 from api.models.user import User
+from api.schemas.user import UserSchema
+
 
 router = Blueprint('registerUser', __name__)
 
 
 @router.route('/register', methods=['POST'])
-def registerUser():
-    try:
-        data = request.get_json()
-    except Exception as e:
-        return jsonify(
-            Response(500).exceptWithMessage(
-                str(e),
-                'Unable to get json'))
-
+def register_user():
+    schema = UserSchema()
+    input_data = request.get_json()
+    data, err = schema.load(input_data)
+    if err:
+        return err
```

Comment:
@gabru-md This will return an error to the console, but what will the user see? Please make use `Response` class and return proper error to the user endpoint

Reply:
I will iterate over this PR.
So incase you are worried about the error stuff, this is just a basic implementation.
Since it works and @yashLadha said it is important, therefore I have given a basic setup so that the frontend can be linked.

If you have any other questions I would suggest waiting for a day or two, so that I can work with @ParthS007 and @yashLadha.

Reply:
@vaibhavsingh97 #848 can you refer to the comments there, cheers :+1: